master/buildbot/newsfragments/render-github-change-hook-secret.bugfix

```diff
@@ -0,0 +1 @@
+The GitHub change hook secret is now rendered correctly.
```

Comment:
Can we reword the release note to something like `The GitHub change hook secret can now be rendered`? It's not that the rendering was incorrect, it was not supported at all.

Reply:
Thanks! Fixed the news fragment as well as renaming this file from `.bugfix` to `.feature`.